### PR TITLE
Create go.yml file to to use GitHub Actions for CI

### PR DESCRIPTION
**Pull request by danrice-square**
CircleCI is disabled due to security concerns (and hasn't been working in this repo for some time). This is a first attempt to build and test using GitHub Actions. It is difficult to test this functionality in a local environment, so some tweaking may be necessary.

### CODE REVIEW

**Comment by danrice-square** [comment]
Tests are passing now. The trick is to forward the mysql port into the mysql container:

```
ports:
          - 3306:3306
```

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      with:
+        go-version: 1.19
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: |
+        export MYSQL_HOST=localhost
+        export MYSQL_PASSWORD=root
```
Comment (bmish) [inline .github/workflows/go.yml:41]: I think you can use `env` for variables? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env

**Comment by bmish** [comment]
Can we delete the Circle CI files and references?

File: .github/workflows/go.yml (hunk 1 above)
Comment (danrice-square) [reply .github/workflows/go.yml:41]: The editor gives my error highlights when I use `env` but maybe it's just a problem with the syntax highlighting. I'll give it a try with `env`.

**Pull request closed without merging**